core/lib/spree/app_configuration.rb

```diff
@@ -426,6 +426,13 @@ def default_pricing_options
     #   Spree::OrderCancellations.
     class_name_attribute :order_cancellations_class, default: 'Spree::OrderCancellations'
 
+    # Allows providing your own class for adjusting items with cancellation adjustments
+    #
+    # @!attribute [rw] cancelled_items_adjuster
+    # @return [Class] a class with the same public interfaces as
+    #   Spree::OrderCancellations::CancelledItemsAdjuster.
+    class_name_attribute :cancelled_items_adjuster, default: 'Spree::OrderCancellations::CancelledItemsAdjuster'
```

Comment:
It would be nice to have a versioned preference here and to change this behavior only for new stores. Unfortunately, this is still not possible for `class_name_attribute`.